src/Runner.Worker/ActionManager.cs

```diff
@@ -378,7 +378,7 @@ public Definition LoadAction(IExecutionContext executionContext, Pipelines.Actio
                 string dockerFileLowerCase = Path.Combine(actionDirectory, "dockerfile");
                 if (File.Exists(manifestFile) || File.Exists(manifestFileYaml))
                 {
-                    var manifestManager = HostContext.GetService<IActionManifestManager>();
+                    var manifestManager = HostContext.GetService<IActionManifestManagerWrapper>();
```

Comment:
Now uses a wrapper class, which performs the comparison depending on the feature flag state